broker/config/crds/backup.servicefabrik.io_v1alpha1_defaultbackups.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
just a question , why is there a `k8s` in this version name ? it will be tied to a platform then ?

Reply:
as they are custom resource definitions, they fall under apiextensions.k8s.io/v1beta1 api, which is part of the apiserver. The name is k8s, which cannot be changed. However, it does not have any affinity to the platform as such. Its how the resource names are defined in apiserver.

Reply:
ok , fine.